packages/compiler/src/server/serverlib.ts

```diff
@@ -1089,6 +1089,21 @@ export function createServer(host: ServerHost): Server {
           };
           host.log(sLog);
         },
+        trackAction: async (asyncAction: () => Promise<any>, startMessage: string) => {
+          host.log({
```

Comment:
can you check this is not overlogging, because we run the emitters consistently

Reply:
this is definitively over logging, removing it